cfscrape/__init__.py

```diff
@@ -58,23 +58,25 @@
 https://github.com/Anorov/cloudflare-scrape/issues\
 """
 
-class ExcludeCaptchaProvokingCiphersAdapter(HTTPAdapter):
-    url = "https://"
+# Remove a few problematic TLSv1.0 ciphers from the defaults
+DEFAULT_CIPHERS += ":!ECDHE+SHA:!AES128-SHA"
```

Comment:
Since the modified ciphers are only used within CloudflareAdapter I think it would be nicer having them as class variables instead of a global.

Reply:
The default headers and user agent is only used within the CloudflareScraper class. It's a default an as such I thought it would nice to expose it.

One use case is changing the defaults.
```py
import cfscrape
cfscrape.DEFAULT_CIPHERS += "!SHA"
```

Another is using them in a custom adapter.
As a class property, it only serves to nest the still very accessible value.
It's a constant in all caps so I'm not worried about global pollution.
I do think it would be better to move this functionality into it's own file as noted in #248 which addresses a lot more than a single global.

Reply:
I understand that scenario but am not sure if it's something we would want to support since it might break the "captcha adapter"

Reply:
I don't think non-advanced user would make use of it and that the same arguments can be made for the other defaults and the adapter class itself.

Reply:
I'm trying to help a user who just opened an issue and was like, du, the DEFAULT_CIPHERS are exposed right now anyway and adding them as a class property won't change that. 

```py
from cfscrape import DEFAULT_CIPHERS
```